conf.py

```diff
@@ -185,7 +185,10 @@
 def setup(app):
 # add_stylesheet() was renamed to add_css_file() in sphinx 1.8 released
 # in September 2018. add_stylesheet() will be removed in sphinx 4.0
-   app.add_stylesheet("sof-custom.css")
+    try:
```

Comment:
duplicate of previous PR ?

Reply:
Sorry, yes, I have based it on PR #345 in order to verify the build, just a sec, I'll drop that patch.

Reply:
Removed the sphinx 4.0 support patch from this PR.